src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -1158,6 +1158,11 @@ public static FormulaValue Exp(IRContext irContext, NumberValue[] args)
 
         public static FormulaValue Power(IRContext irContext, NumberValue[] args)
         {
+            if (args.Length < 2 || args[0] == null || args[1] == null)
+            {

```

Comment:
How is this possible? Does binder guarantee these are not-null?

Reply:
As Greg's mentioned in description of issue: "However, since the Call node is inserted directly by the IR Translator, the standard function processing, like inserting blank checks into the IR, is skipped." so we need to check blank/null in direct calls.